cluster/examples/kubernetes/ozone/install-operator-rbac.yaml

```diff
@@ -0,0 +1,114 @@
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: objectstores.ozone.rook.io
+spec:
+  group: ozone.rook.io
+  names:
+    kind: ObjectStore
+    listKind: ObjectStoreList
+    plural: objectstores
+    singular: objectstore
+  scope: Namespaced
+  version: v1alpha1
```

Comment:
It would be good to include the schema of the CRD settings for validation, but this could be a future PR.

Reply:
Do you mean to include all the possible key/values but commented out or do you talk about a more formal schema definition? I tried to find a good example in the current source but didn't find. Please share an existing example and will copy/follow the pattern in a next PR.

Reply:
Do you mean to include all the possible key/values but commented out or do you talk about a more formal schema definition? I tried to find a good example in the current source but didn't find. Please share an existing example and will copy/follow the pattern in a next PR.